### DISCUSSION TITLE
grapesjs-tools does not follow canva width

### DISCUSSION THREAD

**Discussion by liviacorradini** [GrapesJS/grapesjs#5082] [Q&A] [answered]
I am creating a custom page builder which includes a side menu that expands and collapses. As a result, grapesjs-tools are not functioning properly with the components. Do you have any suggestions on how to resolve this issue?

https://user-images.githubusercontent.com/97858789/235250887-3683cdbe-d09d-4657-8b77-1ead47f442da.mov

**Comment by artf** [comment, marked as answer]
You can trigger [`editor.refresh()`](https://grapesjs.com/docs/api/editor.html#refresh) on your side menu toggle
